src/background/index.ts

```diff
@@ -203,9 +204,6 @@ browser.runtime.onInstalled.addListener(({ reason }) => {
       throw new Error('popupUrl not found');
     }
     const url = new URL(browser.runtime.getURL(popupUrl));
-    url.searchParams.append('templateType', 'tab');
-    browser.tabs.create({
-      url: url.toString(),
-    });
+    openOnboarding(url);
```

Comment:
This was the cause of the main problem. We didn't add context param here, so session was erased on page reload on initial onboarding open